custodia/message/kem.py

```diff
@@ -373,7 +373,8 @@ def test_1_Parse_GET(self):
         kem = KEMHandler({'KEMKeysStore': self.kk})
         kem.parse(jtok, "mykey")
         out = kem.reply('output')
-        jtok = JWT(jwt=json_decode(out)['value'])
+        msg = json_decode(json_encode(out))
```

Comment:
why this decode/encode dance ?
is it just to validate that the output of kem.parse can be encoded in json ?